### DISCUSSION TITLE
Display balloon in  onCreate function of a fragment

### DISCUSSION THREAD

**Discussion by gulcesenyuz** [skydoves/Balloon#435] [Q&A] [answered]
Hi, I created balloon builder in a fragment and also, i can display it when i click an item in fragment.
But i have been trying to call display balloon function in onCreate method of fragment. 
Is there any possible way to initialize it ?

My purpose: "balloon should be displayed after i navigate to X activity for the first time when i open the app"

**Comment by skydoves** [comment, marked as answer]
Hey @gulcesenyuz, the balloon must be initialized by an anchor base, so you need a `View` to display your balloon which means you can display your balloon at least `onViewCreated` in your fragment.

**Reply by skydoves** [reply]
If you want to display a balloon for the first time, you should use the function below:

```kotlin
your_view. showAlignTop(balloon)

or

your_view.post {
  balloon. showAlignTop(your_view)
}
```
